strings/boomer.json

```diff
@@ -0,0 +1,52 @@
+{
+    "boomer": [
```

Comment:
"You need to pull yourself up by the bootstraps and (get a good job, start investing your money, make your credit score great early)."

Reply:
Add "build your 401k for retirement". I hear that one a lot.

Reply:
yes, 401k, how could I forget

Reply:
need something about "buying everything on a credit card so you're used to paying off your debts on time" as well, classic boomer advice right there